app/grants/templates/grants/shared/active_clr_round.html

```diff
@@ -0,0 +1,21 @@
+<div class="container-fluid grants-container">
+  <div class="container">
+    {% for active_round in active_rounds %}
+      <a  href="{% url "grants:clr_grants" round_num=active_round.round_num %}"  class="d-block text-decoration-none position-relative my-3 text-white" style="background: url('{% if active_round.logo %} {{ active_round.logo.url }} {% endif %}') no-repeat center ">
+        <div class="position-absolute" style="top: 0; right: 0; width: 100%; height: 100%; background: #0044e088; z-index: 5;"></div>
+        <div class="d-flex px-4 py-3 justify-content-between position-relative" style="z-index: 10">
+          <div class="d-flex">
+            <div class="bg-white mr-3 p-2 overflow-hidden" style="width: 56px; height: 56px; border-radius: 100px;">
+              <img class="img-responsive" style="width: 100%; height: 100%;" src="/dynamic/avatar/{{ active_round.owner.handle | default:'gitcoinco'}}" />
+            </div>
+            <h5 class="font-weight-bold mb-0"> {{ active_round.display_text |default:active_round.round_num }} <br/>
```

Comment:
@thelostone-mc do you think `sub_round_slug` should replace `display_text`?

Reply:
Ah good catch! I think the same

Reply:
oh i get... default works as round_num then... cause sub_round_slug is optional